cypress/e2e/signIn.cy.js

```diff
@@ -1,7 +1,24 @@
 /// <reference types="cypress" />
 
-describe('Sign In page', () => {
-  it('should provide an ability to log in', () => {
+const logUsername = 'dsarvaaav';
+const email = 'fdsfaasd13as@gmail.com';
+const password = 'dsadfgafsa';
 
+describe('my firs test', () => {
+  it('should', () => {
+    cy.visit('https://conduit.mate.academy/');
+
+    cy.get(':nth-child(3) > .nav-link').click();
```

Comment:
Using CSS selectors like ':nth-child(3) > .nav-link' can be brittle if the structure of the page changes. Consider using more stable selectors, such as data attributes, if available.